src/citrine/_session.py

```diff
@@ -128,5 +128,8 @@ def checked_delete(self, path: str) -> dict:
         return self.checked_request('DELETE', path)
 
     def checked_get(self, path: str, *args, **kwargs) -> dict:
-        """Execute a GET request to a URL and utilize error filtering on the response."""
+        """Execute a GET request to a URL and utilize error filtering on the response.
+
+        :rtype: object
```

Comment:
Suggest deleting unless it can be made more specific (especially since it contradicts the typed return value of `dict`)

Reply:
done